e2e_test/test/e2e_test.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2017, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+void main() {
+  group('setup test', () {
```

Comment:
The advantage is we can run these tests just like any other (using `pub run test`). It should also generally be less fragile (don't have to deal with exit codes explicitly etc), and its generally cleaner for the same reason.

I explicitly set this up so the ordering would not matter any more, the other tests run `dazel init` in their `setUp` method.